facade/relayerFacade.go

```diff
@@ -32,13 +49,22 @@ func (rf *relayerFacade) PprofEnabled() bool {
 }
 
 // GetPeerInfo returns a P2PPeerInfo value holding an unknown peer value
-func (rf *relayerFacade) GetPeerInfo(pid string) ([]core.QueryP2PPeerInfo, error) {
+func (rf *relayerFacade) GetPeerInfo(pid string) ([]elrondCore.QueryP2PPeerInfo, error) {
 	return nil, errors.New("not implemented")
 }
 
-// GetClientInfo returns specified client`s info
-func (rf *relayerFacade) GetClientInfo(client string) (string, error) {
-	return "", errors.New("not implemented")
+// GetMetrics returns specified metric info
+// if the provided name is empty, it will return a list of all available metrics
```

Comment:
I don't like this approach. By calling the same endpoint you can get 2 tot semantic-different responses. Please define a new endpoint `.../list` for example that will list all the status handler available (might be done now, or in a future PR)

Reply:
will be done in the next PR, thanks